File: Javascript/1018.js
var input = `1257`;
var lines = input.split("\n");
var num = lines[0];
let billetes = [100, 50, 20, 10, 5, 2, 1];
console.log(num);

for (let i in billetes) {
  console.log(
    parseInt(num / billetes[i]) + " nota(s) de R$ " + billetes[i] + ",00"
  );
  num %= billetes[i]; 
}

File: Kotlin/1018.kt
import java.util.Scanner

fun main() {
    val input = Scanner(System.`in`);
    var num = input.nextInt();
    val billetes = arrayOf(100, 50, 20, 10, 5, 2, 1);
    println(num);
    for (i in billetes.indices) {
        println("${num / billetes[i]} nota(s) de R$ ${billetes[i]},00");
        num %= billetes[i];
    }
    
}